kernel/dvb_hdhomerun_fe.c

```diff
@@ -103,7 +103,10 @@ static int dvb_hdhomerun_fe_read_ucblocks(struct dvb_frontend* fe, u32* ucblocks
 	return 0;
 }
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,3,0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,9,0)
```

Comment:
```suggestion
#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,6,0)
```
Update this to the exact kernel version that required this change.
These changes in general fix issue h0tw1r3/dvbhdhomerun#8